Project.toml

```diff
@@ -1,27 +1,28 @@
 name = "Bootstrap"
 uuid = "e28b5b4c-05e8-5b66-bc03-6f0c0a0a06e0"
-authors = "Julian Gehring"
-version = "2.1.0"
 license = "MIT"
+authors = "Julian Gehring"
```

Comment:
What is the effect of changing the order here?

Reply:
It doesn't have any effect. That's just the order `Pkg` creates as soon as you activate the project and add or remove a dependency (not sure anymore if that's needed at all or if activating it is already sufficient.